src/components/core/ContextSelector/index.tsx

```diff
@@ -103,7 +99,7 @@ const ContextSelector: React.FC = () => {
                 </>
             );
         },
-        [queryText, currentContext]
+        [queryText, currentContext, dropdownSections]

```

Comment:
This fixes the problem with the item selection. 
causing dropdownsections to update so selectedItem dosnt run with memoized sections at line 57